{
  runtimeService.startProcessInstanceByKey("uncaughtErrorOnCallActivity");
  Task task=taskService.createTaskQuery().singleResult();
  assertEquals("Task in subprocess",task.getName());
  try {
    taskService.complete(task.getId());
  }
 catch (  BpmnError e) {
    assertTextPresent("No catching boundary event found for error with errorCode 'myError', neither in same process nor in parent process",e.getMessage());
  }
}
